pkg/skuba/development_constants.go

```diff
@@ -20,6 +20,7 @@
 package skuba
 
 const (
-	BuildType       = "development"
-	ImageRepository = "registry.suse.de/devel/caasp/5/containers/containers/caasp/v5"
+	BuildType               = "development"
+	ImageRepository         = "registry.suse.de/devel/caasp/5/containers/containers/caasp/v5"
+	ImageRepositoryTemplate = "registry.suse.de/devel/caasp/{NUMBER}/containers/containers/caasp/{VERSION}"
```

Comment:
I'd suggest using this variable https://github.com/SUSE/skuba/pull/1210/files#diff-63d77c0689b762a1fed6bfd18428bd4eR35-R38 and returns v4 URL if <= 1.17; otherwise, returns v5 URL.
It makes it simpler.

Reply:
hmm.. these are build time constant, and component/addon are generate at run time.

Reply:
For `skuba cluster image` command, it iterates all Kubernetes version and prints out the related component/addon image path, I think it _should_ be doable at runtime.